### PR TITLE
Add installation snippets

### PR DESCRIPTION
**Pull request by kroikie**
Add snippets showing the use of the firebase-installations SDK.

### CODE REVIEW

File: installations/settings.gradle [hunk 1]
```diff
@@ -0,0 +1,2 @@
+include ':app'
+rootProject.name = "My Application"
```
Comment (samtstern) [inline installations/settings.gradle:2]: Don't forget to add this project to the root settings.gradle and the CI scripts.

File: installations/app/src/main/java/com/google/samples/snippet/MainActivity.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import com.google.firebase.installations.InstallationTokenResult;
+
+public class MainActivity extends AppCompatActivity {
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        logInstallationAuthToken();
+    }
+
+    private void logInstallationAuthToken() {
+        FirebaseInstallations.getInstance().getToken(true)
```
Comment (rosariopfernandes) [inline installations/app/src/main/java/com/google/samples/snippet/MainActivity.java:21]:
```suggestion
        FirebaseInstallations.getInstance().getToken(/* forceRefresh */ true)
```

File: installations/settings.gradle (hunk 1 above)
Comment (kroikie) [reply installations/settings.gradle:2]: Done

File: installations/app/src/main/java/com/google/samples/snippet/kotlin/MainActivity.kt [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+class MainActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+        logInstallationAuthToken();
+    }
+
+    private fun logInstallationAuthToken() {
+        // [START get_installation_token]
+        FirebaseInstallations.getInstance().getToken(true)
```
[the remark this answers is not in the document]
Comment (kroikie) [reply installations/app/src/main/java/com/google/samples/snippet/kotlin/MainActivity.kt:18]: Kotlin extensions are not ready yet so I'll apply the same change used for Java.

[the remark this answers is not in the document]
Comment (rosariopfernandes) [reply installations/app/src/main/java/com/google/samples/snippet/kotlin/MainActivity.kt]: Kotlin actually supports [named arguments](https://kotlinlang.org/docs/reference/functions.html#named-arguments) out of the box, no need to implement extensions for it :)

[the remark this answers is not in the document]
Comment (kroikie) [reply installations/app/src/main/java/com/google/samples/snippet/kotlin/MainActivity.kt]: Named arguments are not allowed for non-kotlin functions. So the comment is the best we can do for now.

[the remark this answers is not in the document]
Comment (rosariopfernandes) [reply installations/app/src/main/java/com/google/samples/snippet/kotlin/MainActivity.kt]: Oh, I see

**Review by SENGPASEUTH — APPROVED** [review]

File: installations/settings.gradle (hunk 1 above)
Comment (SENGPASEUTH) [reply installations/settings.gradle:2]: Hi

Comment (SENGPASEUTH) [reply installations/settings.gradle:2]: Hello

File: installations/app/src/main/java/com/google/samples/snippet/MainActivity.java (hunk 2 above)
Comment (SENGPASEUTH) [reply installations/app/src/main/java/com/google/samples/snippet/MainActivity.java]: Hi

Comment (SENGPASEUTH) [reply installations/app/src/main/java/com/google/samples/snippet/MainActivity.java]: He

Comment (SENGPASEUTH) [reply installations/app/src/main/java/com/google/samples/snippet/MainActivity.java]: > ```suggestion
>         FirebaseInstallations.getInstance().getToken(/* forceRefresh */ true)
> ```

**Pull request merged**